tests/apollo/test_skvbc_view_change.py

```diff
@@ -600,6 +600,74 @@ async def test_recovering_of_replicas_with_initiated_view_change(self, bft_netwo
         await bft_network.wait_for_fast_path_to_be_prevalent(
             run_ops=lambda: skvbc.run_concurrent_ops(num_ops=20, write_weight=1), threshold=20)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: c == 0, rotate_keys=True)
+    @verify_linearizability()
```

Comment:
Hi @titusen, this test isn't meant to run as part of regular CI, but rather in manual/nightly runs.

Let's mark it as disabled in CI:
```python
@unittest.skip("edge scenario - not part of CI")
```

Reply:
@titusen , on second thought - if the test is really stable, we can also include it as part of the regular CI!

Reply:
It's not stable. Actually it fails. I need to investigate why.